presto-prometheus/src/test/resources/prometheus-data/up_matrix_response.json

```diff
@@ -0,0 +1,33 @@
+{
+  "status": "success",
+  "data": {
+    "resultType": "matrix",
+    "result": [
+      {
+        "metric": {
+          "__name__": "up",
+          "instance": "localhost:9090",
+          "job": "prometheus"
+        },
+        "values": [
+          [
+            1565962969.044,
+            "1"
+          ],
+          [
+            1565962984.045,
+            "1"
+          ],
+          [
+            1565962999.044,
+            "1"
+          ],
+          [
+            1565963014.044,
+            "1"
+          ]
+        ]
+      }
+    ]
+  }
+}
```

Comment:
no new lines before EOF. Here and in other files like that.

Reply:
Done in 26981f10